services/secondary-storage/server/src/main/java/org/apache/cloudstack/storage/resource/NfsSecondaryStorageResource.java

```diff
@@ -3106,17 +3106,16 @@ protected String parseCifsMountOptions(URI uri) {
             extraOpts.append(name + "=" + nvp.getValue() + ",");
         }
 
-        if (logger.isDebugEnabled()) {
-            logger.error("extraOpts now " + extraOpts);
-        }
+        String extraOptions = extraOpts.toString();
+        logger.error("extraOpts now ", ()->StringUtils.cleanString(extraOptions));
```

Comment:
debug log?

Reply:
hm, decisions, decisions, decisions, ... ;)

maybe even trace, I just copied the original call.

Reply:
Agree with @sureshanaparti. IMO, debug makes more sense in this case.

Reply:
```suggestion
        logger.debug("extraOpts now ", ()->StringUtils.cleanString(extraOptions));
```